### ISSUE TITLE
[Bug]Clarification and Correction for Function List in jax.numpy Namespace

### ISSUE THREAD

**Issue by akshatvishu** [unifyai/ivy#25920]
### Problem Description:
I was going through the `issue_list` : [ Add creation functions to Jax frontend #8431 ](<https://github.com/unifyai/ivy/issues/8431>)  and found out that some of the function present there to implement are actually classes:

### Task present in issue list which are `classes`:

- character #24166 <https://jax.readthedocs.io/en/latest/_autosummary/jax.numpy.character.html#jax.numpy.character>
- complexfloating<https://jax.readthedocs.io/en/latest/_autosummary/jax.numpy.complexfloating.html#jax.numpy.complexfloating.__init__>
- complexWarning <https://jax.readthedocs.io/en/latest/_autosummary/jax.numpy.ComplexWarning.html#jax.numpy.ComplexWarning>
- floating <https://jax.readthedocs.io/en/latest/_autosummary/jax.numpy.floating.html#jax.numpy.floating>
- flexible <https://jax.readthedocs.io/en/latest/_autosummary/jax.numpy.flexible.html#jax.numpy.flexible.__init__>
- generic <https://jax.readthedocs.io/en/latest/_autosummary/jax.numpy.generic.html#jax.numpy.generic.__init__>
- integer <https://jax.readthedocs.io/en/latest/_autosummary/jax.numpy.integer.html>
- number <https://jax.readthedocs.io/en/latest/_autosummary/jax.numpy.number.html#jax.numpy.number.__init__>
- object_  <https://jax.readthedocs.io/en/latest/_autosummary/jax.numpy.object_.html#jax.numpy.object_>

refs:https://github.com/google/jax/blob/main/jax/_src/numpy/lax_numpy.py

Instead what can be done is to make open issue for:
```py
bool_ = _make_scalar_type(np.bool_)
uint4 = _make_scalar_type(dtypes.uint4)
uint8 = _make_scalar_type(np.uint8)
uint16 = _make_scalar_type(np.uint16)
uint32 = _make_scalar_type(np.uint32)
uint64 = _make_scalar_type(np.uint64)
int4 = _make_scalar_type(dtypes.int4)
int8 = _make_scalar_type(np.int8)
int16 = _make_scalar_type(np.int16)
int32 = _make_scalar_type(np.int32)
int64 = _make_scalar_type(np.int64)
float8_e4m3fn = _make_scalar_type(dtypes.float8_e4m3fn)
float8_e5m2 = _make_scalar_type(dtypes.float8_e5m2)
float8_e4m3b11fnuz = _make_scalar_type(dtypes.float8_e4m3b11fnuz)
bfloat16 = _make_scalar_type(dtypes.bfloat16)
float16 = _make_scalar_type(np.float16)
float32 = single = _make_scalar_type(np.float32)
float64 = double = _make_scalar_type(np.float64)
complex64 = csingle = _make_scalar_type(np.complex64)
complex128 = cdouble = _make_scalar_type(np.complex128)
```
similar to [uint16 #5130](<https://github.com/unifyai/ivy/pull/5133/files#diff-c08d1eb3c9ccd5be46d0e98475ab98580b238cb6cbdcd6dde5d2269926dab290>) and [int16 #7677](<https://github.com/unifyai/ivy/pull/7701>)

basically everything that uses `_make_scalar_type` can be made into an issue with the same name and can be added to ivy jax frontend at `creation.py

**Comment by hirwa-nshuti** [comment]
Hey @akshatvishu 

This issue was up on my radar, but I was still caught up in other things on my plate 😂
Will surely get started on fixing this 🤗

Thanks for bringing it up 🫡

**Comment by akshatvishu** [comment]
> Hey @akshatvishu
> 
> This issue was up on my radar, but I was still caught up in other things on my plate 😂 Will surely get started on fixing this 🤗
> 
> Thanks for bringing it up 🫡

Thanks for looking into this @hirwa-nshuti !
